src/fragment-components/a-content-switcher.tsx

```diff
@@ -0,0 +1,227 @@
+			}}
+			template={template} />
+	</>;
+};
+
+export const AContentSwitcherCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}}
+/>;
+
+export const AContentSwitcher = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		{...rest}>
+			<ContentSwitcher
+			size={componentObj.size}
+			selectedIndex={componentObj.selectedIndex}
+			className={cx(preventCheckEvent, componentObj.cssClasses?.map((cc: any) => cc.id).join(' '))}>
+			{
+				componentObj.items.map((step: any, index: number) => <Switch
+					className={step.className}
+					name={step.name}
+					text={step.text}
+					disabled={step.disabled}
+					key={index} />
+				)
+			}
+			</ContentSwitcher>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: AContentSwitcher,
+	settingsUI: AContentSwitcherSettingsUI,
+	codeUI: AContentSwitcherCodeUI,
+	keywords: ['content', 'switcher'],
+	name: 'Content switcher',
+	type: 'content-switcher',
+	defaultComponentObj: {
+		type: 'content-switcher',
+		size: 'sm',
+		selectedIndex: 0,
+		items: [
+			{
+				name: 'first',
+				text: 'First section',
+				disabled: false,
+				type: 'switch-item'
+			},
+			{
+				name: 'second',
+				text: 'Second section',
+				disabled: false,
+				type: 'switch-item'
+			},
+			{
+				name: 'third',
+				text: 'Third section',
+				disabled: false,
+				type: 'switch-item'
+			}
+		]
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: (_) => '',
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}Selected = new EventEmitter();`,
+			imports: ['ContentSwitcherModule'],
+			code: ({ json }) => {
+				return `<ibm-content-switcher
+					${angularClassNamesFromComponentObj(json)}
+					(selected)="${nameStringToVariableString(json.codeContext?.name)}Selected.emit()">
+					${json.items.map((step: any, index: number) => `<button
+							${json.selectedIndex === index ? `active="${json.selectedIndex === index}"` : ''}
+							ibmContentOption
+							name="${step.name}"
+							[disabled]="${step.disabled}">
+							${step.text}
+						</button>`
+					).join('\n')}
+				</ibm-content-switcher>`;
+			}
+		},
+		react: {
+			imports: ['ContentSwitcher', 'Switch'],
+			code: ({ json }) => {
+				const name = nameStringToVariableString(json.codeContext?.name);
+				return `<ContentSwitcher
+					size="${json.size}"
+					selectedIndex={${json.selectedIndex}}
+					${reactClassNamesFromComponentObj(json)}
+					onChange={(selectedItem) => handleInputChange({
+						target: {
+							name: "${name}",
+							value: selectedItem
+						}
+					})}>
+					${json.items.map((step: any) => `<Switch
+						name={"${step.name}"}
```

Comment:
```suggestion
						name="${step.name}"
```

Having the curly braces seems a bit redundant to me since the props will be considered a string by default.